Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
+      if File.directory?(f)
+        false
+      else
+        f = File.basename(f)
+        !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
+      end
```

Comment:
`collection.any? { some_code }` passes each element of the collection to the block, and returns true if the block ever returns a value other than false or nil, and false if it does not.

Since there isn't any control flow in either branch of this conditional, the values are being thrown away, and the block will always return whatever `problem` returns, which is always an array (i.e. truthy).

So what is going on here?